packages/libs/web-common/src/components/DatasetGraph.jsx

```diff
@@ -9,6 +9,7 @@ import { safeHtml } from '@veupathdb/wdk-client/lib/Utils/ComponentUtils';
 import ExternalResource from './ExternalResource';
 import { JbrowseIframe } from './JbrowseIframe';
 import { EdaScatterPlot } from './eda/EdaScatterPlot';
+import WdkService from '@veupathdb/wdk-client/lib/Service/WdkService';
```

Comment:
Hi @ryanrdoherty 
The problem here is mainly that `WdkService` is a type, not an object, but because this is a JSX file we don't get any help from the IDE :-( 
I'll see if there's a way to get hold of the actual wdkService object.

Reply:
I don't think there's a way*, partly because it's a class-based component, and partly because the calls to dataPlotter are outside the regular WDK ecosystem, so the WDK hasn't been wired into this component.

(* well, not without extensive refactoring, of course)

Reply:
Yeah I think once it's a functional component `useWdkService` should be usable? Though probably `useWdkEffect` is the more appropriate hook.

The appropriate context is "provided" here:
https://github.com/VEuPathDB/web-monorepo/blob/526752faa0268931f6c3d821b6c9617bf49643d6/packages/libs/wdk-client/src/Core/Root.tsx#L163